### ISSUE TITLE
[KTable]: Shift + Tab navigation is trapped

### ISSUE THREAD

**Issue by AlexVelezLl** [learningequality/kolibri-design-system#883]
## Product
Noticed in Kolibri https://github.com/learningequality/kolibri/pull/12952.

## Expected behavior
When I am already in the first cell of the table. Shift + Tab should have the default behaviour and focus the previous focusable element in the page.

## Actual behavior
Right now, If the first cell of the headers, or if the first cell of the body have a checkbox (or probably any other focusable element inside), `Shift + Tab` is trapped between the cell and the checkbox inside, and we can't focus the previous focusable element outside of the table.

https://github.com/user-attachments/assets/b9c7225b-20da-47ce-b8da-fbcc36da176e

## Steps to reproduce the issue
1. Checkout https://github.com/learningequality/kolibri/pull/12952.
2. Go to Coach > Quiz > Create new Quiz > Select groups or individual learners recipients
3. Keyboard-navigate the Side Panel with Tab and Shift + Tab until you get to the learners table

**Comment by Pandaa007** [comment]
Hello @AlexVelezLl I would really like to work on this issue please assign it to me.

**Comment by AlexVelezLl** [comment]
Hey @Pandaa007! Thanks for your interest in contributing to Kolibri! Will assign this to you :), please let us know in case you have any question 👐

**Comment by Pandaa007** [comment]
Thanks @AlexVelezLl for assigning this issue to me. Surely I will discuss my doubts here.

**Comment by Pandaa007** [comment]
Hi @AlexVelezLl I've made a PR resolving this issue, Please review it.

**Comment by MisRob** [comment]
Hi @Pandaa007, thank you, we will have a look
